pkg/templates/aws/edges/load_balancer-load_balancer_listener.yaml

```diff
@@ -2,12 +2,6 @@ source: aws:load_balancer
 target: aws:load_balancer_listener
 deployment_order_reversed: true
 operational_rules:
-  - if: '{{eq (fieldValue "Type" .Source) "application"}}'
-    configuration_rules:
-      - resource: '{{ .Target }}'
-        configuration:
-          field: Protocol
-          value: HTTP
```

Comment:
do we have a rule somewhere to have the listener track the protocol of the LB?

Reply:
A downstream target group will match the protocol of the listener, which I believe is existing behavior. That said, I'm not sure I agree with it since it's kind of up to the user to decide where they want to terminate SSL.